src/cli.rs

```diff
@@ -204,4 +202,34 @@ mod tests {
         assert!(parse_args(&["supply-chain", "supply-chain", "json", "--print-schema"]).is_err());
         assert!(parse_args(&["supply-chain", "supply-chain", "crates", "-d"]).is_err());
     }
+
+    #[cfg(unix)]
+    fn write_updated(new_val: &str, path: impl AsRef<std::path::Path>) -> std::io::Result<bool> {
```

Comment:
It would be really nice to have a comment explaining how to run this to regenerate documentation.

Reply:
And also, what .tpl even is - I have never heard of that extension.

Reply:
Will include better docs once we decide to do the docs part. `.tpl` short for template. The way it works - you update the parser and run tests, tests update the documentation and fail if detect changes - this way README stays always up to date - otherwise CI fails.